StyleCop.Analyzers/StyleCop.Analyzers.CodeGeneration/RoslynHashCode.cs

```diff
@@ -0,0 +1,462 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// <auto-generated/>
+
+#nullable enable
+
+// NOTE: This code is derived from an implementation originally in dotnet/runtime:
+// https://github.com/dotnet/runtime/blob/v5.0.3/src/libraries/System.Private.CoreLib/src/System/HashCode.cs
```

Comment:
Could you reference [Microsoft.Bcl.HashCode](https://www.nuget.org/packages/Microsoft.Bcl.HashCode) instead?

Reply:
I prefer to avoid dependencies in the source generator, since it makes build integration much more complex.